ttnn/cpp/ttnn/operations/ccl/line_all_gather/device/ccl_line_all_gather_op.hpp

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/operations/ccl/line_all_gather/device/line_all_gather_op.hpp"
+#include "ttnn/cpp/ttnn/multi_device.hpp"
+
+namespace ttnn {
+namespace operations {
+namespace ccl {
+
+struct ExecuteLineAllGather {
```

Comment:
this file should be in line_all_gather folder, one level up and should be called `line_all_gather.hpp`